README.md

```diff
@@ -12,6 +12,20 @@ This is the home of 18F's evolving brand assets and style guide. It's a work in
 ## Coming soon:
 - More templates (Google slides, Good docs, & many more)
 - Better guidance on how to use colors, fonts, logos, and other design assets
-- and more! We're adapting to our team's needs as we go, so if there's something you'd like to see us build or offer guidance on, file an issue in this repository. 
+- and more! We're adapting to our team's needs as we go, so if there's something you'd like to see us build or offer guidance on, file an issue in this repository.
 
+## Contributing
 
+### Installation
+You should just be able to copy and paste each of the following steps to get the brand site up and running.
+
+0. `git clone git@github.com:18F/brand.git && cd brand`
+0. `bundle install`
+0. `cd web-design-standards && git submodule init && git submodule update && cd ..`
+0. `bundle exec jekyll serve`
+0. You should be able to see the branding site at [http://127.0.0.1:4000](127.0.0.1:4000)
+
+### Making changes
+Yay, that's exciting that you want to contribute! Follow the installation instructions to get started but add the `--watch` option to the fourth step so you'd run `bundle exec jekyll serve --watch`. Jekyll is now instructed to watch for changes in your site and then rebuild when it notices them. This helps for local development because it means you can save a change and then refresh in your browser to see it.
+
```

Comment:
@jeremiak what types of changes will `--watch` look for? The reason I ask is I'm only running `jekyll serve` to serve the project and Sass is already re-compiled when I save a change.